testing/internal/e2e/host/static/connect_test.go

```diff
@@ -0,0 +1,72 @@
+package static_test
```

Comment:
This was originally `statichost_test.go`. Not sure why this doesn't show up as a move like some of the other file shere.